pkg/migrations/op_create_table.go

```diff
@@ -404,8 +404,10 @@ func (w *ConstraintSQLWriter) WriteForeignKey(referencedTable string, referenced
 	if w.Name != "" {
 		constraint = fmt.Sprintf("CONSTRAINT %s ", pq.QuoteIdentifier(w.Name))
 	}
-	constraint += fmt.Sprintf("FOREIGN KEY (%s) REFERENCES %s (%s) MATCH %s ON DELETE %s ON UPDATE %s",
-		strings.Join(quoteColumnNames(w.Columns), ", "),
+	if len(w.Columns) != 0 {
```

Comment:
Under what circumstances would we have 0 columns here?

Reply:
In case of inline FK constraints. I am adding a comment so it's clear.